tests/unit/FilingHistoryList.spec.ts

```diff
@@ -7,22 +7,22 @@ import flushPromises from 'flush-promises'
 import axios from '@/axios-auth'
```

Comment:
This file is still under construction (while you review the rest of the changes :) ).

It's because I disabled a bunch of these tests that the code coverage is low for this module.